packages/story-editor/src/app/highlights/quickActions/useQuickActions.js

```diff
@@ -67,8 +68,8 @@ const {
 } = Icons;
 
 export const MediaPicker = ({ render, ...props }) => {
+  const allowedTranscodableMimeTypes = TRANSCODABLE_MIME_TYPES;
```

Comment:
This assignment is a bit pointless :-)

Reply:
I'll fix these up later today ...

To clarify in these case use TRANSCODABLE_MIME_TYPES direct or ?

Reply:
Yes